### PR TITLE
Release v.1.1.0

### PR DESCRIPTION
**Pull request by Shashank-KanakapuraSrivatsa**
This is an initial release, which will subsequently be followed by enhancement to the existing functionalities and also bug fixes. This version includes features such as :

Search Page with search functionality
Search page consists of query box where users can type in the query manually.
Search page has query creation menu which helps a user to build the query by selecting values for specific parts of the query.
Tooltips on metrics to give a short description.
A tabular display of results.
A link in the results that leads to More Information page. This page gives additional information regarding an artifact. The values of all the defined metrics are displayed for an artifact.
Input validations
Error handling
Adoption to newer versions of the API.

### CODE REVIEW

**Comment by bhermann** [comment]
Please do not make major version increments. I think 1.0.0, 1.1.0., ... is bold enough :)

**Comment by Shashank-KanakapuraSrivatsa** [comment]
> 1.0.

Version set to 1.1.0

File: conf/application.conf [hunk 1]
```diff
@@ -349,6 +349,16 @@ db {
   #default.logSql=true
 }
 
-app.portWebapp = 8085
+pidfile.path = "/dev/null"
```
Comment (bhermann) [inline conf/application.conf]: Why?

File: build.sbt [hunk 2]
```diff
@@ -1,18 +1,21 @@
-import com.typesafe.config._
 
+import com.typesafe.sbt.packager.MappingsHelper.directory
+
+
+mappings in Universal ++= directory(baseDirectory.value / "public")
 name := "delphi-webapp"
 
 version := "1.0.0-SNAPSHOT"
```
Comment (bhermann) [inline build.sbt]: When you do a release you should set version number.

File: conf/application.conf (hunk 1 above)
Comment (chinmayk1613) [reply conf/application.conf]: Removed the path reference(No impact on application).

File: build.sbt (hunk 2 above)
Comment (chinmayk1613) [reply build.sbt]: Version number set to 1.1.0

**Comment by chinmayk1613** [comment]
Final version set...

**Pull request merged**